### DISCUSSION TITLE
Rocket.js

### DISCUSSION THREAD

**Discussion by zurtri** [django-commons/django-unicorn#216] [Q&A]
Hi Adam,

I have just completed my first unicorn component.

When I pushed it to production - it couldn't find the rocket.js file - event after a `python manage.py collectstatic`

I found that changing the config setting `MINIFIED` to `False` and then a collectstatic after that made the rocket.js file suddenly be found and the component worked as expected.

Just though your should know!

**Comment by adamghill** [comment]
Where is `rocket.js` located on the filesystem? I assume you load it from a directory with other static assets (e.g. css, images, etc?). The `MINIFIED` setting should only apply to serving `unicorn.min.js` or all of the modules that make up the minified file separately. 

If you want it might be useful to post a screenshot of the developer toolbar that details the 404 loading the js and maybe console error output if there is anything there. I'm just surprised that setting is affecting your code in any way.

**Reply by zurtri** [reply]
Ok got it sussed.

Rocket.js comes from the cloudflare rocket loader.

For some reason Cloudflare wasnt passing all the JS across (or didnt see to build it or something), so I simply turned off Rocket loader.

This seems to have resolved the problem.

**Reply by adamghill** [reply]
Awesome, glad you figured it out!
